Environment/NoiseLevelObserved/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Environment/NoiseLevelObserved/schema.json",
+  "title": "GSMA / FIWARE - Noise Level observed schema",
+  "description": "An observation of those acoustic parameters that estimate noise pressure levels at a certain place and time. ",
+  "type": "object",
+  "allOf": [
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons" },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "NoiseLevelObserved"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "dateObserved": {
+          "type": "string"
+        },
+        "dateObservedFrom": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "dateObservedTo": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "refDevice": {
+          "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"
+        },
+        "refPointOfInterest": {
+          "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"
+        },
+        "refWeatherObserved": {
```

Comment:
weather? not documented in the specs.

also, sonometerClass is missing.

Reply:
yup, thanks for noticing